### ISSUE TITLE
Submit to pangeo-forge

### ISSUE THREAD

**Issue by martindurant** [lsterzinger/fsspec-reference-maker-tutorial#3]
Now that [HDFReferenceRecipe](https://github.com/pangeo-forge/pangeo-forge-recipes/blob/master/pangeo_forge_recipes/recipes/reference_hdf_zarr.py#L25) is available in pangeo-forge, it should be possible to submit a new recipe repeating the work in this tutorial to https://github.com/pangeo-forge/staged-recipes

It would be nice to also do the NWM, but that would take some work on the recipe class to support the two-stage aggregation required by the large number of files.

**Comment by lsterzinger** [comment]
What's the difference between https://github.compangeo-forge/staged-recipes and https://github.com/pangeo-forge/pangeo-forge-recipes?

**Comment by martindurant** [comment]
1. pangeo-forge-recipes is the library containing the classes that get called during the run phase, when a dataset is processed.
2. staged-recipes is where individual datasets are defined via PRs (like in conda-forge), spawning one new repo per dataset. These recipes provide arguments to be passed to the classes in 1, in a yaml file and/or python code.

Think of it like fsspec-reference-maker's classes in 1., and each specific example that creates a JSON as 2.

**Comment by lsterzinger** [comment]
Great. I want to work on getting the HRRR/GRIB2 stuff added as well. So if I understand it correctly, the GOES/NetCDF stuff will go in staged-recipes since you did the work of putting SingleHdf5ToZarr in pangeo-forge-recipes, while GRIB2 examples would require additions to both repos.

**Comment by martindurant** [comment]
Yes, that's correct - the class currently in pangeo-forge-recipes only deals with HDF
